test/functions/math.cpp

```diff
@@ -382,3 +382,14 @@ TEST_CASE("lcm()of big integers", "[functions][math][lcm][big]") {
         "533220692127153044311875258011747917053108027629278373174251200266431"
         "428784066739966");
 }
+
+TEST_CASE("Base cases for is_probable_prime()", "[functions][math][is_probable_prime]") {
+	BigInt num = 1;
+	REQUIRE(num.is_probable_prime(25) == 1);
+	num = 2;
+	REQUIRE(num.is_probable_prime(25) == 1);
+	num = 3;
+	REQUIRE(num.is_probable_prime(25) == 1);
+	num = 5;
+	REQUIRE(num.is_probable_prime(25) == 1);
```

Comment:
I can't test because Binary Arithmetic Tests are failing 
![image](https://user-images.githubusercontent.com/13573860/46910275-8f462c80-cf0f-11e8-8c68-49ab40c14d96.png)

Reply:
This is because `a` is not declared. I think you forgot to replace all instances of `a` to `rand_num`.